stablehlo/reference/Ops.cpp

```diff
@@ -386,15 +405,8 @@ SmallVector<InterpreterValue> eval(Region &region,
       scope.add(complexOp.getResult(), result);
     } else if (auto compositeOp = dyn_cast<CompositeOp>(op)) {
       auto operands = scope.findTensors(compositeOp.getOperands());
-      SmallVector<InterpreterValue> values = llvm::to_vector(llvm::map_range(
-          operands, [](const Tensor &t) { return InterpreterValue(t); }));
-      SymbolTableCollection symbolTableCollection;
-      auto symbolTable =
-          symbolTableCollection.getSymbolTable(op.getParentOfType<ModuleOp>());
-      auto func = symbolTable.lookupNearestSymbolFrom<func::FuncOp>(
-          &op,
-          StringAttr::get(op.getContext(), compositeOp.getDecomposition()));
-      auto results = eval(func.getBody(), values, fallback, process, nullptr);
+      auto results = evalCallOp(operands, fallback, process, &op,
```

Comment:
Would be cool in the future to enhance `fallback` to be able to specify supported composites, then this function would actually behave like the impl to composite -- is there special handling? Use it. Else call decomp.

Reply:
Ooh yeah that's a cool idea! I'll keep that in mind.